docs/book/v3/reference/migration.md

```diff
@@ -0,0 +1,17 @@
+# Migration to version 3.0
+
+However, starting in this version, we offer a few changes affecting the
+following that you should be aware of, and potentially update your application
+to adopt:
+
+- [PHP 7.2 support](#php-7.2-support)
+- [Strict typing](#strict-typing)
```

Comment:
Please remove these links because the documentation theme creates already a table of contents with links.

Example: https://docs.laminas.dev/laminas-config/intro/

Reply:
Removed.